src/latticpp/marshal/gohandle.h

```diff
@@ -177,20 +183,29 @@ namespace latticpp {
     using PublicKey = GoHandle<GoType::PublicKey>;
     using Plaintext = GoHandle<GoType::Plaintext>;
     using Ciphertext = GoHandle<GoType::Ciphertext>;
+    using CiphertextQP = GoHandle<GoType::CiphertextQP>;
     using RotationKeys = GoHandle<GoType::RotationKeys>;
     using SwitchingKey = GoHandle<GoType::SwitchingKey>;
```

Comment:
An idea is to have a `typedef` defining `RotationKey` (singluar) to be the same as `SwitchingKey`. Depends on the decision regarding SwitchingKey/RotationKey in the discussion above.

Reply:
Yes, not sure about that, your call